quickevent/app/plugins/qml/Event/src/Event/eventconfig.cpp

```diff
@@ -106,13 +106,13 @@ void EventConfig::save(const QString &path_to_save)
 			val_str = val.toString();
 		q_up.bindValue(":key", key);
 		q_up.bindValue(":val", val);
-		q_up.exec();
+		q_up.exec(qf::core::Exception::Throw);
```

Comment:
`q_up.exec(!qf::core::Exception::Throw);` 

is intention here, there is no try-catch block in function call stack, so I prefer not crashing app with error in log in this case

If we want to enable throw exception here, we should change more code in other places